utils/android/README.md

```diff
@@ -0,0 +1,16 @@
+
+# Build Android Toolchain
+
+This toolchain will generate the .so and .swiftmodule files of swift standard library and Foundation framework for Android environment, armv7 architecture. Those files are needed when building any swift library to be included in an applciation for Android.
```

Comment:
Please capitalize "Swift" and correct the typo in the word "application".
Insert "the" before "Swift standard library" and before "Android environment".